Allow mobile exporter to export string arrays

pushkartripathi opened the pull request:
Summary: Allow mobile exporter to export string arrays

Differential Revision: D9061213

pjh5 commented:
Retrying build but saving old status https://ci.pytorch.org/jenkins/job/pytorch-builds/job/pytorch-macos-10.13-py3-trigger/14641/

pjh5 commented:
Two more retries on master
https://ci.pytorch.org/jenkins/job/pytorch-builds/job/pytorch-macos-10.13-py3-trigger/14829/
https://ci.pytorch.org/jenkins/job/pytorch-builds/job/pytorch-macos-10.13-py3-trigger/14832/

pushkartripathi committed:
Allow mobile exporter to export string arrays (#10017)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/10017

Allow mobile exporter to export string arrays

Differential Revision: D9061213

fbshipit-source-id: 4e32eebe034b72e55eb0f8a364f0c21953c74e3e
